### ISSUE TITLE
Option to play audio coordinate when using object navigation

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2559]
**Reported by jteh on 2012-07-25 07:05**
Sometimes, it is useful to get a quick idea of where an object is located on the screen. While mouse audio coordinates can be used to do this, this could be extended so that an audio coordinate beep is heard when using object navigation indicating the location of the current navigator object. This would be particularly useful for touch screen users when flicking between objects.
**Blocking #4072**

**Comment by josephsl** [comment]
Hi,

Coming back to this issue...

about to rebase it based on the current master branch and moving the branch to my own forked repo so people can start testing this.

A possible follow-up issue might be asking NVDA to play coordinates when one uses focus command such as tab, arrows and so on (for this, an even simpler solution is to expand what becomeNVDAObject event can do by playing coordinate beeps).

Thanks.

**Comment by josephsl** [comment]
Hi,

Blocked by #5906 due to the following:

If indent announcement is set to beep, moving to an edit field that indicates indents (such as Notepad++) will mean one of the new beeps will not play. Will wait until a decision on what to give priority is made.

Thanks.

**Comment by derekriemer** [comment]
@josephsl @MichaelDCurran I'm curious if simply playing indent beeps 80 MS or so in the future would help alleviate this bug

**Comment by josephsl** [comment]
Hi,

Coming back to this...

Speech refactor may open up possibilities for this work, although lately I've been thinking about withdrawing this issue in favor of Audio Themes or similar mechanism.

Thanks.

**Comment by Brian1Gaff** [comment]
Yes I find the 3D nature of that add on very handy. For example when a 
little alert from dropbox comes along you can hear its that from the sound 
style and the fact that it is on one channel.

It might make sence for those wanting to get some idea of where something is 
on a screen from left to right and up and down if some form of theme or 
pitch could be introduced for the screen location of a control.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by josephsl** [comment]
Hi,

2024 update: with the introduction of sound split, I don't think we can proceed with this issue unless it is okay to play audio coordinates for navigator objects across channels. But then it may cause issues for folks using mono audio. Therefore, I'm closing the issue for now.

Thanks.

**Comment by Adriani90** [comment]
The doubt related to sound split is not valid, because the same principle applies for already implemented mouse coordinates. By the way, sound split can be disabled completely in NVDA if needed.
I am reopening.
